near-contract-standards/src/fungible_token/mod.rs

```diff
@@ -7,5 +7,8 @@ pub mod receiver;
 pub mod resolver;
 pub mod storage_impl;
 
+pub use self::core::FungibleTokenCore;
+pub use crate::storage_management::StorageManagement;
 pub use core_impl::FungibleToken;
 pub use macros::*;
+pub use resolver::FungibleTokenResolver;
```

Comment:
Let's align on one way of importing things. I like `self::`, but it is not the style used before, so let's not use it now.

Reply:
It is the same way we re-export in `non_fungible_token`. The reason is that we expose `pub mod core` several lines above, so we can't use `crate` anymore.

Reply:
But in fact since we only have one entity there - we can just re-export that directly.

Reply:
Agreed.

Reply:
There is a reason I used `self` there and it has to do with the fact that otherwise we get an ambiguity error. I have changed that to use the fully qualified path `crate::fungible_token`..

Reply:
Note, that we do use `self` for `non_fungible_token` imports. For that same reason - there is a name ambiguity problem with `core`.

Reply:
@ruseinov We should consider migrating all the imports to use `self::` or `crate::` for local imports, but it is a separate discussion.

Reply:
`self` is interesting for disambiguation scenarios, because the path ends up being shorter.